remappings.txt

```diff
@@ -1,6 +1,11 @@
-ds-guard/=lib/ds-guard/src/
+ds-math/=lib/ds-math/src/
 ds-test/=lib/ds-test/src/
 ds-token/=lib/ds-token/src/
 ds-auth/=lib/ds-token/lib/ds-auth/src/
-ds-math/=lib/ds-token/lib/ds-math/src/
```

Comment:
Not sure what is the best practice here, but we can optimize this like: https://github.com/ClioFinance/socgen-ofh-onboarding-prototype/pull/3/files#diff-6ea402a2ca81b3b6f5dba5eed6bfac5e5d8bf8660be870732ad6e202e1038a68R1

Reply:
What do you mean exactly? I don't get it.